Add services documentation to Picnic docs (#21944)
home-assistant/home-assistant.github.io: source/_integrations/picnic.markdown

| Next delivery ETA end          | End of the ETA window of the next delivery. |
| Next delivery slot start       | Start of the next delivery's delivery slot. |
| Next delivery slot end         | End of the next delivery's delivery slot. |

## Services

### Service `picnic.add_product`

> Add a product to you cart using the `picnic.add_product` service, either using a product ID or a product name.

Reviewer: ```suggestion
Add a product to your cart using the `picnic.add_product` service, either using a product ID or a product name.
```

A search will be done and the first result will be added to the cart when one adds a product using a product name.

> The service call will fail when no product could be found or when no `product_id` or `product_name` were specified. 

Reviewer: ```suggestion
The service call will fail when no product can be found, or when no `product_id` or `product_name` is specified. 
```
